src/CodingTheory.jl

```diff
@@ -31,6 +32,7 @@ import Oscar.GAP: GapObj, Globals, Packages
 import Base: circshift, iseven, show, length, in, zeros, ⊆, /, *, ==, ∩, +, -, copy, isequal, ∘
 import Combinatorics: powerset
 import DataStructures: capacity
+import Hecke: GroupAlgebraElem, FinGenAbGroupElem, FinFieldElem, CTFieldElem, fpFieldElem
```

Comment:
should also remove. Note that anything beginning with CT is defined by this package